www-src/app/replicator/change/change_event_handler.coffee

```diff
@@ -1,28 +1,68 @@
+AndroidCalendarHandler = require "../../lib/android_calendar_handler"
+CozyToAndroidEvent = require "../transformer/cozy_to_android_event"
 log = require('../../lib/persistent_log')
     prefix: "ChangeEventHandler"
     date: true
 
 module.exports = class ChangeEventHandler
 
-    change: (doc) ->
-        log.info "change"
+    constructor: (@calendarSync) ->
+        @androidCalendarHandler = new AndroidCalendarHandler()
+        @cozyToAndroidEvent = new CozyToAndroidEvent()
+        @calendarSync = navigator.calendarsync unless @calendarSync
```

Comment:
Could you add a light comment to explain why you add calendarSync as parameter ?  (to allow mock in test I suppose).

Personally, I would write (but it's just code style) :
```
constructor: (calendarSync) -> 
....
@calendarSync = navigator.calendarsync unless calendarSync
```